pkg/hostagent/hostagent.go

```diff
@@ -128,7 +128,8 @@ func New(instName string, stdout io.Writer, sigintCh chan os.Signal, opts ...Opt
 		}
 		vSockPort = port
 	} else if *y.VMType == limayaml.QEMU {
-		virtioPort = filenames.VirtioPort
+		// virtserialport doesn't seem to work reliably: https://github.com/lima-vm/lima/issues/2064
```

Comment:
Could you report this to https://gitlab.com/qemu-project/qemu/-/issues ?

Reply:
Not sure what exactly I would report; I have no more information beyond that the code in Lima sometimes fails. It doesn't even fail reliably, which is why the bisect script restarts the VM up to 100 times before accepting that the code works properly.

I haven't checked if the problem happens on the sending or the receiving side, as we are not getting any errors.

So I feel this would need some more investigation before reporting to upstream, but unfortunately I'm out of time right now. 😞